starters/nuxt3-pinia-vuetify/components/CounterExample/CounterExample.vue

```diff
@@ -0,0 +1,61 @@
+<script lang="ts" setup>
+import { useCounterStore } from '@/store/counterStore';
+
+const counterState = useCounterStore();
```

Comment:
I think this should be named `counterStore` rather than `counterState` since it is a reference to the store (of which the store does contain the state but it also can contain getters and actions)